vars/malwareExecuteScan.groovy

```diff
@@ -0,0 +1,42 @@
+import groovy.transform.Field
+
+import com.sap.piper.GenerateDocumentation
+
+@Field String STEP_NAME = getClass().getName()
+
+@Field STEP_CONFIG_KEYS = [
+
+
+    /**
+      * Malware scanning host.
+      */
+    'host',
+    /**
+      * The file which is scanned for malware
+      */
+    'file',
+    /**
+      * The credentials used for accessing the malware scanning host
+      */
+    'credentialsId'
+
+]
+
+@Field GENERAL_CONFIG_KEYS = STEP_CONFIG_KEYS
+
+@Field PARAMETER_KEYS = GENERAL_CONFIG_KEYS
+
+@GenerateDocumentation
+void call(Map parameters = [:]) {
+
+    handlePipelineStepErrors (stepName: STEP_NAME, stepParameters: parameters) {
+
+        def cred = [
+            type: 'usernamePassword',
+            id: 'credentialsId',
```

Comment:
please apply name according to 
https://github.com/SAP/jenkins-library/blob/master/.github/CONTRIBUTING.md#jenkins-credential-handling

Reply:
done